app/src/common/components/MovieList/MovieElement/Title/Title.jsx

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import './Title.scss';
+
+export default class Title extends React.Component {
```

Comment:
Can be a functional stateless component.

Reply:
done